### PR TITLE
[RM Ch09] Automation section moved to chapter end

### PR DESCRIPTION
**Pull request by walterkozlowski**
This is in a partial response to Issue #3088 "Cloud Infrastructure Lifecycle automation; chapter restructure"

Move Telemetry and Observability section to be after Assurance section because these topics are related; improve wording of the opening sentence of the Telemetry and Observability section as suggested by @pgoyal01 
Move Automation discussion to the end of Chapter 9, as this content is somewhat self-contained

### CODE REVIEW

**Comment by walterkozlowski** [comment]
@joaoarodrigues, can you add a remark, and I will try to add you as a reviewer

File: doc/ref_model/chapters/chapter09.rst
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter09.rst:228]: 1. "demanding and challenging" are synonyms
2. task or job (consists of multiple tasks)

File: doc/ref_model/chapters/chapter09.rst
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter09.rst:229]: "... continually being increased ..." sounds very odd. since "complexity is used later in the sentence, may I suggest (not the best) to replace "being increased" with "becoming more so"

Comment (pgoyal01) [inline doc/ref_model/chapters/chapter09.rst:229]: "... production excellence" can mean different things to different folks.
What does "production excellence" look like in the cloud era or Kubernetes? Both recognise failure.

File: doc/ref_model/chapters/chapter09.rst
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter09.rst:236]:
```suggestion
a system with a remarkably high level of complexity that requires significant effort for managing and operating
```

File: doc/ref_model/chapters/chapter09.rst
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter09.rst:244]:
```suggestion
Common support system (such as, OSS – Operation Support System, BSS – Business Support System) requirements are redefined,
```

File: doc/ref_model/chapters/chapter09.rst
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter09.rst:239]:
```suggestion
A large and complex ecosystem of end-user services requires a formalized approach
```

File: doc/ref_model/chapters/chapter09.rst
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter09.rst:240]:
```suggestion
for achieving high reliability and scalability. The discipline which applies well-known practices of
```

**Review by pgoyal01 — COMMENTED** [review]
will review Observability later

**Comment by joaoarodrigues** [comment]
> @joaoarodrigues, can you add a remark, and I will try to add you as a reviewer

--> Reviewer profile

File: doc/ref_model/chapters/chapter09.rst
Comment (walterkozlowski) [reply doc/ref_model/chapters/chapter09.rst]: @pgoyal01, please check the new wording

File: doc/ref_model/chapters/chapter09.rst
Comment (walterkozlowski) [reply doc/ref_model/chapters/chapter09.rst]: @pgoyal01, please check the new wording

Comment (walterkozlowski) [reply doc/ref_model/chapters/chapter09.rst]: @pgoyal01, please check the new wording

**Comment by walterkozlowski** [comment]
@pgoyal01, @karinesevilla, @ulikleber, @jbhartley, @joaoarodrigues - please review this PR and approve if you can.  The purpose of this PR was to move Automation section to the end of the chapter, as agreed.  Once approved and merged, I am going to review the contents, align to Ch 3 and 8, and polish up wording.

**Review by ulikleber — COMMENTED** [review]
Why is the first part ("Telemetry and Observability") not only moved but largely re-worded? This makes reviewing difficult. Also the PR description doesn't mention any re-wording.

**Review by pgoyal01 — APPROVED** [review]

**Comment by joaoarodrigues** [comment]
> @pgoyal01, @karinesevilla, @ulikleber, @jbhartley, @joaoarodrigues - please review this PR and approve if you can. The purpose of this PR was to move Automation section to the end of the chapter, as agreed. Once approved and merged, I am going to review the contents, align to Ch 3 and 8, and polish up wording.

@walterkozlowski I'm not in the Reviewer's list. Could you please check it?

**Review by jbhartley — APPROVED** [review]
@walterkozlowski @pgoyal01 @joaoarodrigues @ulikleber  I am providing review of this based on "section move".

I believe this needs a readability review as the language is difficult and jargonistic and could be greatly simplified.

**Comment by jbhartley** [comment]
@joaoarodrigues to your comment on "I'm not in the Reviewer's list. Could you please check it?".

I believe is you just post a comment, then you should automatically then pop up in the reviewer list. I have had this problem before. I was not able to be put into reviewer list, but once I made a comment, then it all worked ok.

**Comment by joaoarodrigues** [comment]
> @joaoarodrigues to your comment on "I'm not in the Reviewer's list. Could you please check it?".
> 
> I believe is you just post a comment, then you should automatically then pop up in the reviewer list. I have had this problem before. I was not able to be put into reviewer list, but once I made a comment, then it all worked ok.

It didn't happen. Could you try to add me as a reviewer?

thank you

**Review by karinesevilla — APPROVED** [review]
I approve the move, but I think that the section Telemetry and Observability must be include into the section Assurance. The two sections are linked: "Assurance" is the goal, "Telemetry and Observability" the means. 

@karinesevilla, I agree.  This will be a subject of another PR.

**Pull request merged**
